components/org.wso2.carbon.identity.oauth.dcr.endpoint/src/gen/java/org/wso2/carbon/identity/oauth2/dcr/endpoint/dto/RegistrationRequestDTO.java

```diff
@@ -51,6 +52,12 @@ public void setClientName(String clientName) {
     this.clientName = clientName;
   }
 
+  @ApiModelProperty
+  @JsonProperty("aud")
```

Comment:
the newly introduced needs to be added as a property of Registration Request Object [1] of the swagger API definition for the endpoint.

[1] https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/components/org.wso2.carbon.identity.oauth.dcr.endpoint/src/main/resources/api.identity.oauth.dcr.endpoint.yaml#L264